### PR TITLE
Rewrite of TabbedAppDrawerFragment.

### PR DESCRIPTION
**Pull request by stanipintjuk**
Devided it into 2 different classes, and added a class for tab
and the fragment information.
Most important thing to note is that tab id and the tab index are no longer
related, so it should be easer to add and remove tabs.

Would be nice if e.g. @XyLoNaMiyX could take a quick glance and tell me if a missed something.

### CODE REVIEW

**Comment by Lonami** [comment]
Yay! Testing time! :) Let's see let's see.

File: app/src/main/java/com/launcher/silverfish/AppDrawerTabFragment.java [hunk 1]
```diff
@@ -161,7 +161,7 @@ private void loadApps() {
         i.addCategory(Intent.CATEGORY_LAUNCHER);
 
         switch (tabId) {
-            case 0:
+            case 1:
                 // Tab 0 is a special tab and includes all except for the once in other tabs
```
Comment (Lonami) [inline app/src/main/java/com/launcher/silverfish/AppDrawerTabFragment.java]: This should say "Tab 1" @stanipintjuk ;)

**Comment by Lonami** [comment]
You've done a good job. It's much cleaner now! And I'm glad you fixed those annoying parts which sometimes added or subtracted 1 magically.

**Comment by stanipintjuk** [comment]
Woops :P 

Yeah it was really annoying and confusing. I have no idea what I was thinking when I originally wrote it

**Pull request merged**